gravitee-am-management-api/gravitee-am-management-api-rest/src/main/resources/views/default/registration_confirmation.html

```diff
@@ -113,7 +113,7 @@
 <![endif]-->
 <script th:src="@{assets/material/material.min.js}"></script>
 <script th:src="@{assets/js/jquery-3.5.1.min.js}"></script>
-<script th:src="@{assets/js/password-validation.js}"></script>
+<script th:src="@{assets/js/password-validation-v2.js}"></script>
```

Comment:
This javascript location`assets/js` is from `gravitee-am-ui/src/assets/js`. To align with the change we need to rename:
from: `gravitee-am-ui/src/assets/js/password-validation.js` 
to: `gravitee-am-ui/src/assets/js/password-validation-v2.js`